tst/com/amazon/corretto/crypto/provider/test/AesTest.java

```diff
@@ -579,6 +581,24 @@ public void test_initParameters() throws Throwable {
         assertArrayEquals(PLAINTEXT, decrypted);
     }
 
+    @SuppressWarnings("ConstantConditions")
+    @Test
+    public void test_initNullKey() throws Throwable {
+        jceC.init(Cipher.ENCRYPT_MODE, key);
+
+        Key key = null;
```

Comment:
NitPick: Variables should be `final` if possible.

Reply:
Fixed.